apps/demo/src/pages/examples/form-component.astro

```diff
@@ -36,7 +36,9 @@ const nameForm = new FormGroup(
       value: "Doe",
     },
   ],
-  "Name"
+  {
```

Comment:
This change should not be needed when you make the new config object optional :)

Reply:
That mean we can accept both `string` for form name and a `FormConfig` for form configuration, right?